connectivity/drivers/ble/FEATURE_BLE/TARGET_NORDIC/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+add_library(mbed-os-ble-nordic-cordio OBJECT)
+
 mbed_add_cmake_directory_if_labels("TARGET")
```

Comment:
Can you explain why you did 
```
if("CORDIO_LL" IN_LIST MBED_TARGET_LABELS)
    add_subdirectory(TARGET_CORDIO_LL)
endif()
```
but in many other places you are still using `mbed_add_cmake_directory_if_labels()`